tools/install_fbthrift.sh

```diff
@@ -17,40 +17,24 @@ if installCheck "$0"; then
   exit 0;
 fi
 
-git clone https://github.com/facebook/fbthrift.git
-cd fbthrift/thrift/
-git checkout d811b530a4f5e11a520f5fb416a5a3a8a5f42ef8
-./build/deps_ubuntu_14.04.sh
-cd ../
-git clone https://github.com/facebook/folly.git
-cd folly/folly
-git checkout 6645b95d6402b48e7b3abdc01146a7a61a9378c9
-autoreconf -ivf
-./configure
-make
-make check
-sudo make install
-cd ../
-# rm -rf .git
-cd ../
-git clone https://github.com/facebook/wangle.git
-cd wangle/wangle
-git checkout 86c4794422e473f3ed5b50035104e1bc04c9646d
-cmake . -DBUILD_TESTS=OFF
-make
-ctest
-sudo make install
-cd ../
-# rm -rf .git
-cd ../
-cd thrift/
-autoreconf -if
-./configure
-sudo make
-sudo make install
-cd ..
-# rm -rf .git
-cd ..
+if [ ! -d fbthrift ]; then
+  git clone https://github.com/facebook/fbthrift.git
+  if [ $? -ne 0 ]; then
+    echo "Could not clone FBThrift!!! Please try again later..."
+    exit 1
+  fi
+fi
+
+cd fbthrift/thrift \
+ && git checkout b2c1f8ed2937c04d9d7de6b07fc6303aec67fb46 \
+ && echo "d6cd4a4a1502a57022ecfd83a988b87512613c06" > ./build/FOLLY_VERSION \
+ && echo "cfb38af8c1e4b27e4405c3953212379b13521e5a" > ./build/WANGLE_VERSION \
+ && ./build/deps_ubuntu_14.04.sh \
```

Comment:
@KamalGalrani, what gcc version are you using on your 14.04? 
I am still getting that error from a previous comment:
`You were able to successfully build on 16.04 without checking out git checkout d811b530a4f5e11a520f5fb416a5a3a8a5f42ef8? On 14.04, I get an error folly/FBVector.h:1432:49: error: parameter packs not expanded with '...' without this checkout.` 
I found a [source](https://github.com/facebook/mcrouter/issues/191) that says `we need at least gcc-4.9 to compile folly`.